deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/QAProvisioning.py

```diff
@@ -9,17 +9,53 @@
 
 
 class QAProvisioning():
-
-    def __init__(self, infra_obj):
-        self.infra_obj = infra_obj
+    """Class to control different options and instances to provisioning with Wazuh and QA Framework.
+
+    Attributes:
+        provision_info (dict): Dict with all the info needed coming from config file.
+        instances_list (list): List with every instance (each host) needed to build the ansible inventory.
+        group_dict (dict): Dict with groups and every host belonging to them.
+        host_list (list): List with every host given in config file.
+        inventory_file_path (string): Path of the inventory file generated.
+        wazuh_installation_paths (dict): Dict indicating the Wazuh installation paths for every host.
```

Comment:
```suggestion
        wazuh_installation_paths (dict): Dict indicating the Wazuh installation paths for every host.
        
```

Reply:
Done in 
https://github.com/wazuh/wazuh-qa/pull/1715/commits/d2d6f33566218b13fa077808ceece2ea1ff71a50